R-Interface/jasprcpp.cpp

```diff
@@ -158,27 +164,19 @@ void STDCALL jaspRCPP_init(const char* buildYear, const char* version, RBridgeCa
 	rInside[".baseCitation"]		= CSTRING_TO_R(baseCitation);
 	rInside[".resultsFont"]			= resultsFont;
 
-	jaspResults::setSendFunc(sendToDesktopFunction);
-	jaspResults::setPollMessagesFunc(pollMessagesFunction);
-	jaspResults::setBaseCitation(baseCitation);
-	jaspResults::setInsideJASP();
-
-	jaspRCPP_logString("Initializing jaspResultsModule.\n");
-
-	rInside["jaspResultsModule"]			= givejaspResultsModule();
-
-	//Adding some functions in R to the RefClass (generator) in the module
-	//To do: move this entirely to zzzWrapper if this wasn't done yet.
-	jaspRCPP_parseEvalQNT("jaspResultsModule$jaspTable$methods(addColumnInfo = function(name=NULL, title=NULL, overtitle=NULL, type=NULL, format=NULL, combine=NULL) { addColumnInfoHelper(name, title, type, format, combine, overtitle) })");
-	jaspRCPP_parseEvalQNT("jaspResultsModule$jaspTable$methods(addFootnote =   function(message='', symbol=NULL, col_names=NULL, row_names=NULL) { addFootnoteHelper(message, symbol, col_names, row_names) })");
+	rInside[".sendToDesktopFunction"]	= Rcpp::XPtr<sendFuncDef>(&sendToDesktopFunction);
+	rInside[".pollMessagesFunction"]	= Rcpp::XPtr<pollMessagesFuncDef>(&pollMessagesFunction);
+	rInside[".baseCitation"]			= baseCitation;
+	jaspRCPP_parseEvalQNT("jaspResults:::setSendFunc(.sendToDesktopFunction)");
```

Comment:
And this actually works?
As in, jaspResults is still sending stuff afterwards?

Reply:
I tested descriptives + a few other analyses and those worked.